addon/templates/components/polaris-resource-list/bulk-actions/bulk-action-button.hbs

```diff
@@ -1,49 +1,37 @@
 {{#component
-  (if url
+  (if @url
     (component "polaris-unstyled-link"
       class="Polaris-ResourceList-BulkActions__Button"
-      external=external
-      url=url
+      external=@external
+      url=@url
     )
     (component "wrapper-element"
       tagName="button"
       type="button"
-      disabled=disabled
+      disabled=@disabled
       class=(concat
         "Polaris-ResourceList-BulkActions__Button"
-        (if disabled " Polaris-ResourceList-BulkActions--disabled")
+        (if @disabled " Polaris-ResourceList-BulkActions--disabled")
       )
-      click=(action onAction)
+      click=(fn (or @onAction this.onAction))
     )
   )
-  ariaLabel=accessibilityLabel
-  onMouseUp=handleMouseUpByBlurring
+  ariaLabel=@accessibilityLabel
+  onMouseUp=this.handleMouseUpByBlurring
 }}
-  {{#component
-    (if disclosure
-      (component "wrapper-element"
-        tagName="span"
-        class="Polaris-ResourceList-BulkActions__ActionContent"
-      )
-      (component "wrapper-element"
-        tagName=""
-      )
-    )
-  }}
-    {{#wrapper-element
-      tagName=(if disclosure "span" "")
-    }}
-      {{#if hasBlock}}
-        {{yield}}
-      {{else}}
-        {{render-content content}}
-      {{/if}}
-    {{/wrapper-element}}
-
-    {{#if disclosure}}
-      <span class="Polaris-ResourceList-BulkActions__ActionIcon">
-        {{polaris-icon source="caret-down"}}
-      </span>
-    {{/if}}
```

Comment:
Have we lost this block somehow? 😕

Reply:
What the dang heck. Not sure what happened, will add back 😳